lib/match.rb

```diff
@@ -0,0 +1,147 @@
+require_relative "board"
+require_relative "player"
+
+class Match
+
+
+    def initialize(board)
+
+        @player_1 = Player.new("X")
+        @player_2 = Player.new("O")
+        @turn_type = decide_turns
+        @board = board 
+        @game_over = false 
+        @moves = []
```

Comment:
When we say `@moves` here, what do we mean?

Reply:
To store the moves of the game. Every time a player has played a turn then that move is stored in that array. Why match? Because match is the class that controls the turns of the players, mostly controls the game.